src/controllers/processors/UserEmailFormSubmissionProcessor.ts

```diff
@@ -42,6 +43,11 @@ export class UserEmailFormSubmissionProcessor implements FormSubmissionProcessor
             .map(data => data[APPLICATION_DATA_KEY] as ApplicationData)
             .unsafeCoerce();
 
-        await this.emailService.send(buildEmail(userProfile, applicationData.appeal));
+        const email = buildEmail(userProfile, applicationData.appeal);
+
+        loggerInstance()
```

Comment:
I find this useless since email service already does `loggerInstance().debug('${EmailService.name} - send: message=${JSON.stringify(message)}')`. Log will be almost same in both places. Please remove it from here and from other processor that sends emails.

Reply:
removed log in EmailService in favour for the log here